Adafruit_Thermal.cpp

```diff
@@ -81,11 +81,13 @@ void Adafruit_Thermal::timeoutSet(unsigned long x) {
 // This function waits (if necessary) for the prior task to complete.
 void Adafruit_Thermal::timeoutWait() {
   if (dtrEnabled) {
-    while (digitalRead(dtrPin) == HIGH)
-      ;
+    while (digitalRead(dtrPin) == HIGH) {
+      delay(0);
```

Comment:
can you make these delay(1) or yield()?

Reply:
Sure, can bump up value. Was avoiding `yield()` based on this comment:
https://github.com/adafruit/Adafruit-Thermal-Printer-Library/pull/26#issuecomment-304168712
Idea being that ESP8266's `delay()` calls `yield()` under the hood. Hopefully using `delay()` is more universal.

Reply:
yield is most like delay(0), delay(1) of course delays. ill say i use yield() in libraries and never heard a complaint

Reply:
it may be old lore? all the BSPs have caught up now? `yield` makes more sense than `delay`, was just wanting to avoid preprocessor logic if possible.

Reply:
yeah all BSPs have caught up, that was 4 years ago :D

Reply:
works4me